administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -7,9 +7,24 @@ COM_JOOMLAUPDATE_CAPTIVE_HEADLINE="Confirm your credentials"
 COM_JOOMLAUPDATE_CHECKED_UPDATES="Checked for updates."
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_DESC="Shows the checkbox to confirm you have taken a backup and you're ready to update in the final step before the update is actually applied."
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_LABEL="Confirm Backup Checkbox"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_DESC="How much of the update ZIP archive to download on each page load. Higher values are faster, but may time out on slow servers. Higher values are safer from timeouts, but the update downloads slower. If the download hangs try using 1 MiB or lower values."
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_DESC="How much of the update ZIP archive to download on each page load. Higher values are faster, but may time out on slow servers. Lower values are safer from timeouts, but the update downloads slower. If the download hangs try using 1 MiB or lower values."
```
I think that could be a copy paste error.